### PR TITLE
User panel component

### PR DESCRIPTION
**Pull request by martadiachok**
Add user panel component; update URL

### CODE REVIEW

**Review by ViktoriaMilkevich — APPROVED** [review]

**Review by Zeicks — APPROVED** [review]

File: src/main/java/com/easyrest/components/UserRoleMenuPanel.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+public class UserRoleMenuPanel {
+    protected WebDriver driver;
+
+    private final By userPanelButton = By.xpath("//ul[@role='menu']//child::a");
+    private final By logOutButton = By.xpath("//ul[@role='menu']//child::li");
+    private final By userProfileIcon = By.xpath("//button[@aria-haspopup='true']//child::div");
+
+    public UserRoleMenuPanel(WebDriver driver) { this.driver = driver; }
+
+    public UserRoleMenuPanel clickOnUserProfileIcon(){
```
Comment (dmatsev) [inline src/main/java/com/easyrest/components/UserRoleMenuPanel.java]: please make this method private

Comment (dmatsev) [reply src/main/java/com/easyrest/components/UserRoleMenuPanel.java]: looks like this comment is obsolete

**Review by vadymtsehelny — APPROVED** [review]

File: src/main/java/com/easyrest/components/UserRoleMenuPanel.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import org.openqa.selenium.WebDriver;
+import com.easyrest.components.Header;
+
+public class UserRoleMenuPanel {
+    protected WebDriver driver;
+
+    private final By userPanelButton = By.xpath("//ul[@role='menu']//child::a");
+    private final By logOutButton = By.xpath("//ul[@role='menu']//child::li");
+
+    Header header = new Header(driver);
```
Comment (dmatsev) [inline src/main/java/com/easyrest/components/UserRoleMenuPanel.java]: The header component is not a part of the menu panel, but the menu panel is a part of the header.

File: src/main/java/com/easyrest/components/Header.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+public class Header {
+    private final By logo = By.xpath("//a[text()='Easy-rest']");
+    private final By homeButton = By.xpath("//a[@aria-current='page']");
+    private final By restaurantListButton = By.xpath("//header[contains(@class,'MuiAppBar-root')]/div/nav/a[2]");
+    private final By userProfileIcon = By.xpath("//button[@aria-haspopup='true']//child::div");
+
+    protected WebDriver driver;
+    public Header(WebDriver driver) { this.driver = driver; }
+
+    protected void clickOnUserProfileIcon(){
```
Comment (dmatsev) [inline src/main/java/com/easyrest/components/Header.java]: Click on user icon should return an instance of menu panel component

Comment (dmatsev) [inline src/main/java/com/easyrest/components/Header.java]: why protected?

File: src/main/java/com/easyrest/components/Header.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        driver.findElement(userProfileIcon)
+                .click();
+    }
+
+    private void clickOnRestaurantList(){
```
Comment (dmatsev) [inline src/main/java/com/easyrest/components/Header.java]: why private?

File: src/main/java/com/easyrest/components/Header.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        driver.findElement(restaurantListButton)
+                .click();
+    }
+
+    private void clickOnHome(){
```
Comment (dmatsev) [inline src/main/java/com/easyrest/components/Header.java]: why private?

File: src/main/java/com/easyrest/components/Header.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+        driver.findElement(homeButton)
+                .click();
+    }
+
+    private void clickOnLogo(){
```
Comment (dmatsev) [inline src/main/java/com/easyrest/components/Header.java]: why private?

**Review by dmatsev — APPROVED** [review]

**Pull request merged**